static/bidder-info/sovrn.yaml

```diff
@@ -17,6 +17,6 @@ userSync:
     url: "https://ap.lijit.com/pixel?redir={{.RedirectURL}}"
     userMacro: "$UID"
   iframe:
-    url: "https://ap.lijit.com/beacon/prebid-server/?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&url={{.RedirectURL}}"
+    url: "https://ap.lijit.com/beacon/prebid-server/?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&url={{.RedirectURL}}"
+    redirectUrl: "{{.ExternalURL}}/setuid?bidder={{.SyncerKey}}&f={{.SyncType}}&uid={{.UserMacro}}"
```

Comment:
I agree with @gargcreation1992 in that I don't see a need to override the `redirectUrl`.

The Redirect URL is for calling back to Prebid Server. Overriding this value to exclude privacy policies may prevent Prebid Server from adhering to regulations. It may be appropriate for host companies to override this field for specific cases or for adapters to cautiously override for highly custom uid scenarios.

Reply:
With the default `redurectUrl` we get the same consent info in the `url` query parameter value and dedicated consent parameters `gdpr`, `gdpr_consent`, `gpp`, `gpp_sid`. So we changed the config to address this duplication. But now I think you're right. We'll handle this on our side and keep configs as close to default as possible.